src/ProjectNeon/Assets/Features/Effects/Events/UnqueueEffectBehavior.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+public class UnqueueEffectBehavior : MonoBehaviour
+{
+    [SerializeField] private BattleState battleState;
+
+    void OnEnable()
+    {
+        BattleEvent.Subscribe<RemoveEffectFromQueue>((effect) => battleState.QueuedEffects.Remove(effect.Effect), this);
```

Comment:
In the lambda, the message isn't the effect. Better to use something more terse, or generic, like `x` or `msg` or `e`...

Reading `effect.Effect` is weird.

Reply:
@SilasReinagel Fixed!